content/en/getting-started/installation.md

```diff
@@ -8,148 +8,190 @@ cascade:
 {{< /alert >}}
+{{< /tab >}}
+{{< /tabpane >}}
 
-Afterwards you should be able to use the LocalStack CLI in your terminal:
-
-{{< command >}}
-$ localstack --help
-Usage: localstack [OPTIONS] COMMAND [ARGS]...
-
-  The LocalStack Command Line Interface (CLI)
-
-Options:
-...
-{{< / command >}}
-
-#### Updates
-
-The LocalStack CLI also allows you to easily update the different components of LocalStack. To check the various options available for updating, run:
-
-{{< command >}}
-$ localstack update --help
-
-Usage: localstack update [OPTIONS] COMMAND [ARGS]...
-
-  Update LocalStack components
-
-Options:
-  --help  Show this message and exit.
-
-Commands:
-  all             Update all LocalStack components
-  docker-images   Update container images LocalStack depends on
-  localstack-cli  Update LocalStack CLI tools
-{{< / command >}}
-
-You can decide to update the CLI itself, the LocalStack Docker images, or all at once:
-
+To verify that the LocalStack CLI was installed correctly, you can check the version in your terminal:
 {{< command >}}
-$ # Update all components
-$ localstack update all
-$ # Only update the LocalStack docker images
-$ localstack update docker-images
-$ # Only update the LocalStack CLI
-$ localstack update localstack-cli
+$ localstack --version
+{{< localstack-latest-version >}}
 {{< / command >}}
 
-#### Starting LocalStack with the LocalStack CLI
+You are all set! Check out our [Quickstart guide]({{< ref "quickstart" >}}) to start your local cloud!
 
-By default, LocalStack is started inside a Docker container by running:
+## Alternatives
+Besides using the CLI, there are other ways of starting and managing your LocalStack instance:
 
-{{< command >}}
-$ localstack start
-{{< / command >}}
+- [LocalStack Cockpit]({{< ref "#localstack-cockpit" >}})\
+  Get a desktop experience and work with your local LocalStack instance via the UI.
 
-{{< alert title="Notes" >}}
-- This command loads all services provided by LocalStack, they will however be started on the first request reaching this service.
+- [Docker]({{< ref "#docker" >}})\
+  Use the `docker` CLI to manually start the LocalStack Docker container.
 
-- By default, LocalStack uses the image tagged `latest` that is cached on your machine, and will **not** pull the latest image automatically from Docker Hub (i.e., the image needs to be pulled manually if needed).
+- [Docker-Compose]({{< ref "#docker-compose" >}})\
+  Use `docker-compose` to configure and start your LocalStack Docker container.
 
-- The default image `localstack/localstack` in DockerHub refers to the community version of LocalStack. The `localstack/localstack-pro` image refers to the Pro version of LocalStack. Previously we maintained `localstack-light` and `localstack-full` images which has been deprecated and will be removed with the LocalStack 2.0 release.
+- [Helm]({{< ref "#helm" >}})\
```

Comment:
Potentially we could also list the [LocalStack Docker Extension](https://docs.localstack.cloud/user-guide/tools/localstack-docker-extension/) as an installation option here.

Reply:
Good point! I added it to the list of alternatives similar to the cockpit.